CHANGES

```diff
@@ -1,6 +1,93 @@
-DD mmm YYYY - 2.9.x (to be released)
+03 Sep 2024 - 2.9.8
 -------------------
 
+ * Fixed ap_log_perror() usage
+   [PR #3241 - @marcstern]
+ * Memory leaks + enhanced logging
+   [PR #3191 - @marcstern]
+ * CI improvement: First check syntax & always display error/audit logs
+   [PR #3190 - @marcstern]
+ * Fixed assert() usage
+   [PR #3202 - @marcstern]
+ * Removed useless code
+   [PR #3193 - @marcstern]
+ * feat: Check if the MP header contains invalid character
+   [PR #3226 - @airween]
+ * Use standard httpd logging format in error log
+   [PR #3192 - @marcstern]
+ * fix msc_regexec() != PCRE_ERROR_NOMATCH strict check
+   [PR #3194 - @marcstern]
+ * Move xmlFree() call to the right place
+   [Issue #3199 - @airween]
+ * Add collection size in log in case of writing error
+   [Issue #3198 - @marcstern]
+ * Passing address of lock instead of lock in acquire_global_lock()
+   [Issue #3188 - @marcstern]
+ * Invalid pointer access in case rule id == NOT_SET_P
+   [Issue #3187 - @marcstern]
+ * Show error.log after httpd start in CI
+   [Issue #3171 - @marcstern]
+ * chore: add pull request template
+   [Issue #3159 - @fzipi]
+ * chore: add gitignore file
+   [Issue #3158 - @fzipi]
+ * Possible double free
+   [Issue #3155 - @marcstern]
+ * Set 'jit' variable's initial value
+   [Issue #3154 - @marcstern]
+ * Missing null byte + optimization
+   [Issue #3153 - @marcstern]
+ * fix: remove usage of insecure tmpnam
```

Comment:
`tmpname`?

Reply:
Thanks - I generated CHANGES through GH API, and picked up descriptions from PR's. Or it can be a single typo by me :smile: